src/functions-reference/mixed_operations.Rmd

```diff
@@ -10,28 +10,57 @@ vector, row vector and arrays.
 Return the matrix m itself.
 `r since("2.3")`
 
+<!-- complex_matrix; to_matrix; (complex_matrix m); -->
+\index{{\tt \bfseries to\_matrix }!{\tt (complex\_matrix m): complex\_matrix}|hyperpage}
+
+`complex_matrix` **`to_matrix`**`(complex_matrix m)`<br>\newline
+Return the matrix m itself.
+`r since("2.30")`
+
 <!-- matrix; to_matrix; (vector v); -->
 \index{{\tt \bfseries to\_matrix }!{\tt (vector v): matrix}|hyperpage}
 
 `matrix` **`to_matrix`**`(vector v)`<br>\newline
 Convert the column vector v to a `size(v)` by 1 matrix.
 `r since("2.3")`
 
+<!-- complex_matrix; to_matrix; (complex_vector v); -->
+\index{{\tt \bfseries to\_matrix }!{\tt (complex\_vector v): complex\_matrix}|hyperpage}
+
+`complex_matrix` **`to_matrix`**`(complex_vector v)`<br>\newline
+Convert the column vector v to a `size(v)` by 1 matrix.
+`r since("2.30")`
+
 <!-- matrix; to_matrix; (row_vector v); -->
 \index{{\tt \bfseries to\_matrix }!{\tt (row\_vector v): matrix}|hyperpage}
 
 `matrix` **`to_matrix`**`(row_vector v)`<br>\newline
 Convert the row vector v to a 1 by `size(v)` matrix.
 `r since("2.3")`
 
-<!-- matrix; to_matrix; (matrix m, int m, int n); -->
-\index{{\tt \bfseries to\_matrix }!{\tt (matrix m, int m, int n): matrix}|hyperpage}
+<!-- complex_matrix; to_matrix; (complex_row_vector v); -->
+\index{{\tt \bfseries to\_matrix }!{\tt (complex\_row\_vector v): complex\_matrix}|hyperpage}
+
+`complex_matrix` **`to_matrix`**`(complex_row_vector v)`<br>\newline
+Convert the row vector v to a 1 by `size(v)` matrix.
+`r since("2.30")`
+
+<!-- matrix; to_matrix; (matrix M, int m, int n); -->
+\index{{\tt \bfseries to\_matrix }!{\tt (matrix M, int m, int n): matrix}|hyperpage}
 
-`matrix` **`to_matrix`**`(matrix m, int m, int n)`<br>\newline
-Convert a matrix m to a matrix with m rows and n columns filled in
+`matrix` **`to_matrix`**`(matrix M, int m, int n)`<br>\newline
```

Comment:
[optional]. I think `A` is better for a matrix, because we use `M` for integer constants.  Also, the textbooks all use `A`, `B`, `C`, etc.